.circleci/config.yml

```diff
@@ -25,17 +25,23 @@ jobs:
       - restore_cache:
           key: v1-node-tslib-deps-{{ checksum "tslib/yarn.lock" }}
       - run: cd admin; yarn install --frozen-lockfile
+      - run:
+          name: Install JUnit coverage reporter
+          command: cd admin; yarn add --dev jest-junit
+      - run:
+          name: Run tests with JUnit as reporter
+          command: cd admin; ./node_modules/.bin/jest --ci --runInBand --reporters=default --reporters=jest-junit --collectCoverage=true
+          environment:
+            JEST_JUNIT_OUTPUT_DIR: "reports/junit/js-test-results.xml"
```

Comment:
reordering the sequence here to optimise for fast failing, so it only performs the install before the test run on each app.